sensor_msgs/Image image
---
string class_id
float32 confidence
